src/coreml.cs

```diff
@@ -15,6 +15,13 @@
 using XamCore.CoreVideo;
 using XamCore.Foundation;
 
+#if !WATCH
+using XamCore.Metal;
+#else
+using IMTLCommandBuffer = XamCore.Foundation.NSObject;
```

Comment:
1. In `scenekit.cs` we add a `global::` prefix, https://github.com/xamarin/xamarin-macios/blob/master/src/scenekit.cs#L65. Not sure if we should add it or not.

2. I'm not sure what's the difference between adding a `using` like you did or adding a stub like here https://github.com/xamarin/xamarin-macios/blob/master/src/scenekit.cs#L98 so the code compiles with `[NoWatch]`. There's a mix of both in `scenekit.cs`.

Reply:
1. @VincentDondain Good Catch!

2. I do not like empty stubs too much since they become part of `temp.dll` and can potentially collide between the different api definition files and would force you to use the full type name in order to avoid a type collision in bindings. Also, I like a lot more the using statements because they will be in a single place (top of the file) and empty stubs can be all over the place.

Reply:
1. Ok cool but can do you know why `global::` is needed? Seems like it was working before.

2. So we could have not used stubs at all in `scenekit.cs` it's totally equivalent to `using` right? I wonder why we used a mix? @spouliot maybe?

Reply:
1. `global::` is just referring to the root namespace, it is just there to avoid possible type clashes, to be honest, I just needed it once when a customer created a class with the same name as one inside our Xamarin.iOS.dll when doing a binding... here is a much better explanation than mine even with an example heh https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/namespaces/how-to-use-the-global-namespace-alias